examples/point_discharge/point_discharge.py

```diff
@@ -0,0 +1,261 @@
+"""
+TELEMAC-2D `Point Discharge with Diffusion' test case
+=====================================================
+
+Solves tracer advection equation in a rectangular domain with
+uniform fluid velocity, constant diffusivity and a constant
+tracer source term. Neumann conditions are imposed on the
+channel walls, an inflow condition is imposed on the
+left-hand boundary, and the right-hand boundary remains open.
```

Comment:
maybe mention here that this is a steady-state problem for which an analytical solution exists